install/5etools-install.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 community-scripts ORG
+# Author: TheRealVira
+# License: MIT
+# Source: https://5e.tools/
+
+# Import Functions und Setup
+source /dev/stdin <<<"$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+# Installing Dependencies with the 3 core dependencies (curl;sudo;mc)
+msg_info "Installing Dependencies"
+$STD apt-get install -y \
+  curl \
+  sudo \
+  git \
+  jq \
+  apache2
+
+msg_ok "Installed Dependencies"
+
+# Setup App
+msg_info "Setup 5etools"
+cat <<EOT >> /etc/apache2/apache2.conf
+<Location /server-status>
+    SetHandler server-status
+    Order deny,allow
+    Allow from all
+</Location>
+EOT
```

Comment:
```suggestion
cat <<EOF > /etc/apache2/apache2.conf
<Location /server-status>
    SetHandler server-status
    Order deny,allow
    Allow from all
</Location>
EOF
```

Move this under Creating Service Section